suite-native/module-send-receive/src/screens/ReceiveScreen.tsx

```diff
@@ -23,16 +25,23 @@ export const ReceiveScreen = ({
     navigation,
 }: StackProps<SendReceiveStackParamList, SendReceiveStackRoutes.Receive>) => {
     const [addressIsVisible, setAddressIsVisible] = useState(false);
-    const { accountKey, tokenSymbol } = route.params;
+    const isScreenFocused = useIsFocused();
 
+    const { accountKey, tokenSymbol } = route.params;
     const account = useSelector((state: AccountsRootState) =>
         selectAccountByKey(state, accountKey),
     );
-
     const token = useSelector((state: AccountsRootState) =>
         selectEthereumAccountToken(state, accountKey, tokenSymbol),
     );
 
+    useEffect(() => {
+        // Hide address when user leaves the screen
+        if (!isScreenFocused) {
+            setAddressIsVisible(false);
+        }
+    }, [isScreenFocused]);
+
```

Comment:
No need. Will be handled in a navigation

Reply:
What do you mean?